client/my-sites/domains/domain-management/edit-contact-info/form-card.jsx

```diff
@@ -145,9 +146,13 @@ class EditContactInfoFormCard extends Component {
 
 	renderTransferLockOptOut() {
 		const { domainRegistrationAgreementUrl, translate } = this.props;
-		const transferLockExpiration = this.props.selectedDomain.transferAwayEligibleAt;
+		const registrationDatePlus60Days = moment
+			.utc( this.props.selectedDomain.registrationDate )
+			.add( 60, 'days' )
+			.toISOString();
```

Comment:
I think this `.toISOString` isn't necessary as Moment methods work with Moment objects too